src/lib.rs

```diff
@@ -60,9 +62,34 @@ pub enum VadTransition {
     SpeechEnd {
         /// When the speech ended, in milliseconds since the start of the VAD session.
         timestamp_ms: usize,
+        /// The active speech samples.
```

Comment:
Probably add a doc comment that with the serde feature enable this field will be skipped